workflows/amr/run.wdl

```diff
@@ -195,24 +199,24 @@ task RunRedup {
             exit 0
         fi
 
-        grep -v "^1\t" "~{cluster_sizes}" | cut -f2 > duplicated-reads.txt
-        grep -h ">" ~{sep=' ' subsampled_reads} | sed "s/^>//" > passed_filters.txt
+        # extract read ids from the subsampled fasta files
+        awk 'index($1, ">") == 1 { print substr($1, 2) }' ~{sep=' ' subsampled_reads} > passed_filters.txt
 
         python3 <<CODE
         pair_values = []
         passed_filters = set()
         duplicates = set()
         with open("passed_filters.txt", "r") as pf:
             passed_filters.update(pf.read().splitlines())
-        with open("duplicated-reads.txt", "r") as dr:
+        with open("duplicated_reads.txt", "r") as dr:
             duplicates.update(dr.read().splitlines())
         with open("~{clusters}", "r") as clusters:
             for line in clusters:
                 key, value = line.strip().split(",")
                 if key in duplicates and key in passed_filters and key != value:
```

Comment:
I know I wrote this line, but I'm a bit dubious that all of it is necessary. If a `key != value` I think that by definition means that it'll be a duplicate. We should merge this in to get the fix out, but possibly consider removing this check entirely.